site/docs/v1/tech/apis/identity-providers/_identity-provider-login-response-body.adoc

```diff
@@ -79,5 +79,6 @@ include::../_user-response-body.adoc[]
 The access token, this string is an encoded JSON Web Token (JWT). This cookie is written in the response as an HTTP Only session cookie.
 
 [field]#refresh_token# [type]#[String]#::
-The refresh token. This cookie is written in the response as an HTTP only persistent cookie. The cookie expiration is configured in the JWT
-configuration for the application or the global JWT configuration.
+The refresh token. This cookie is written in the response as an HTTP only persistent cookie. The cookie expiration is configured in the JWT configuration for the application or the global JWT configuration.
+
+:clearfix!:
```

Comment:
What is this?!?!  Well, adoc is so kind as to continue processing this description list item out of the include and continue adding things to the definition that are after the include.  So this:
```
include::docs/v1/tech/apis/identity-providers/_oauth-idp-operations.adoc[]
:idp_display_name!:
:idp_id!:
:idp_login_method:
:idp_lowercase!:
:idp_since!:
:idp_token_or_code!::idp_type!:
```
Gets rendered like this:
<img width="866" alt="Screen Shot 2021-06-10 at 1 19 56 PM" src="https://user-images.githubusercontent.com/635238/121591789-c3d75280-c9ee-11eb-911c-01623c2ffe86.png">

Reply:
Is `clearfix` an asciidoc primitive? I didn't see it with a brief google.

Reply:
Sorry, `clearfix` is my attempt at a joke.  It doesn't matter what the variable is (it could be `:foobar!:`, `:close_list!`, `:anything!:`, etc) I just need AsciiDoc to end the description list.  Much like we use an `inlcude` for list items like, `debug` for example, so we can make `ifdef` and `ifndef` checks without the list breaking.  Well, here I'm intentionally breaking the list inside the `include::` so the attributes that get undefined just outside the `include::` don't get pulled into the list item.  This way they get processed and undefined.